docs/howtoguides/how_to_not_fix_related_usns.md

```diff
@@ -2,7 +2,7 @@
 
 When running the `pro fix` command for a USN, by default we also try to fix
 any related USNs as well (To better understand the concept of related USNs, please take 
-a look at this [document](.../references/what_are_related_usns.md). To make this clear,
+a look at this [document](../explanations/cves_and_usns_explained.md). To make this clear,
```

Comment:
It won't let me add a suggestion that includes a line that hasn't been edited, but could you please change the text around the link to make a contextual one:

"...related USNs as well. To better understand the concept of related USNs, you can refer to [our related USNs guide](../explanations/cves_and_usns_explained.md). To make this clear..."

Reply:
Excellent point - I'll fix this before merging